whoami/Cargo.toml

```
@@ -0,0 +1,49 @@
+[package]
+name = "whoami"
+version = "2.0.0-pre.2"
+edition = "2021"
+license = "Apache-2.0 OR BSL-1.0 OR MIT"
+documentation = "https://docs.rs/whoami"
+homepage = "https://github.com/ardaku/whoami/releases"
+repository = "https://github.com/ardaku/whoami"
+readme = "README.md"
+description = "Retrieve the current user and environment."
+keywords = ["user", "username", "whoami", "platform", "detect"]
+categories = [
+    "os",
+    "wasm",
+    "internationalization",
+    "localization",
+    "value-formatting",
+]
+include = [
+    "/LICENSE_APACHE",
+    "/LICENSE_BOOST",
+    "/LICENSE_MIT",
+    "/README.md",
+    "/src/*",
```

I'm not sure, you should be able to check with `cargo package`, but this will need to change to either

```suggestion
    "/whoami/src/*",
```

or

```
    "../LICENSE_APACHE",
    "../LICENSE_BOOST",
    "../LICENSE_MIT",
    "../README.md",
    "/src/*",
```

---

Good catch! Thank you!